src/libstd/sys/wasm/process.rs

```diff
@@ -14,12 +14,14 @@ use io;
 use sys::fs::File;
 use sys::pipe::AnonPipe;
 use sys::{unsupported, Void};
+use sys_common::process::{CommandEnv, DefaultEnvKey};
 
 ////////////////////////////////////////////////////////////////////////////////
 // Command
 ////////////////////////////////////////////////////////////////////////////////
 
 pub struct Command {
+    env: CommandEnv
```

Comment:
Should be `CommandEnv<DefaultEnvKey>` here?

```
[00:41:37] error[E0243]: wrong number of type arguments: expected 1, found 0
[00:41:37]   --> /checkout/src/libstd/sys/wasm/process.rs:24:10
[00:41:37]    |
[00:41:37] 24 |     env: CommandEnv
[00:41:37]    |          ^^^^^^^^^^ expected 1 type argument
```

Reply:
Yeah will fix that - ooi, is there a `cargo check` equivalent for `x.py`? I'd like to be able to compile targets like wasm/fuchsia/etc. without needing to have a whole cross-compiling toolchain...

Reply:
> is there a `cargo check` equivalent for `x.py`?

Not yet. Feel free to file a feature-request.

However, doing a `x.py check` on stage(N+1) still requires a complete stageN compiler, meanwhile `--keep-stage` is pretty broken (#44737, #46180), so this may not be as useful as you'd think.

Reply:
Fixed.